cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2020 The Kubeedge Authors.
```

Comment:
can you use the current year with 2021?

Reply:
My fault, it should be 2019.This file was created at that time.

Reply:
Why we have to use the date, when the file was created? Can we used the last modification date instead? And that is in the current year ....

Reply:
`Copyright` usually starts with the first date and ends with the second date.

Reply:
Yes you are right